### ISSUE TITLE
Please upgrade the role to install Forgejo 1.21.6-0

### ISSUE THREAD

**Issue by sergiodj** [elusyx/ansible-role-forgejo#7]
Forgejo 1.21.7-0 contains an important security fix. Please update the role when possible. Thanks!

**Comment by mattredact** [comment]
Done! Waiting on merge into main MASH-Playbook.
